autoscheduler/scraper/views.py

```diff
@@ -14,3 +16,17 @@ def get_object(self):
         course_num = self.request.query_params.get('course_num')
         term = self.request.query_params.get('term')
         return Course.objects.get(dept=dept, course_num=course_num, term=term)
+
+class ListSectionView(generics.ListAPIView):
+    """ API endpoint for viewing course information, used by /api/sections.
+        This view returns a serialized section, should
+        return list of all sections for a given course.
+    """
+    serializer_class = SectionSerializer
+
+    def get_queryset(self):
+        """ Overrides default behavior of get_object() to work without a primary key """
+        dept = self.request.query_params.get('dept')
+        course_num = self.request.query_params.get('course_num')
+        term = self.request.query_params.get('term')
+        return Section.objects.filter(subject=dept, course_num=course_num, term_code=term)

```

Comment:
```suggestion
        return Section.objects.filter(subject=dept, course_num=course_num, 
                                      term_code=term).order_by('id')
```

Although logically I don't think this needs to be ordered, in order to pass the tests you need to make it ordered by the ID (so section ID 1 is before section ID 2 in the output)

Reply:
Ignore this suggestion and see #152, as it removes the need for it.